specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/examples/GetHDInsightCapabilities.json

```diff
@@ -242,7 +242,7 @@
             ]
           }
         },
-        "vmSize_filters": [
+        "vmsize_filters": [
```

Comment:
The same with the property "vmsizes", I know this property should be changed to vmSizeFilters,  but now in the backend, the response is vmsize_filters. I have created a work item to track this.
Our service will GA a new api version in April, in the new api version, I will fix this issue. 
If it is ok, I hope this can check in. @JeffreyRichter You opinion is important for us, I will follow your suggestion. How do you like?